tests_and_analysis/test/euphonic_test/test_spectrum1d.py

```diff
@@ -0,0 +1,223 @@
+import os
+import json
+
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+from euphonic import ureg
+from euphonic.spectra import Spectrum1D
+from tests_and_analysis.test.utils import get_data_path
+
+class ExpectedSpectrum1D:
+    def __init__(self, spectrum1d_json_file: str):
+        self.data = json.load(open(spectrum1d_json_file))
+
+    @property
+    def x_data(self):
+        return np.array(self.data['x_data'])*ureg(
+            self.data['x_data_unit'])
+
+    @property
+    def y_data(self):
+        return np.array(self.data['y_data'])*ureg(
+            self.data['y_data_unit'])
+
+    @property
+    def x_tick_labels(self):
```

Comment:
Given that it would be a bit ugly to have a _single_ type annotation in the module, this could also expressed by adding an explicit
```python
else:
    return None
```

Which is going to happen anyway, but looks like it might have been an oversight with the current setup.